netlify.toml

```diff
@@ -18,6 +18,6 @@
     REACT_APP_API_URL = "https://t-ndo.herokuapp.com"
 
 # Deploy Preview context: All deploys generated from a pull/merge request
-# will inherit these settings. 
+# will inherit these settings.
 [context.deploy-preview.environment]
-  REACT_APP_API_URL = "https://tndo-temp-staging.herokuapp.com"
\ No newline at end of file
+  REACT_APP_API_URL = "https://tieme-ndo-backend-staging1.herokuapp.com"
```

Comment:
Not sure whether the original URL staging should be changed as this is our main one. @Pav0l ?

Reply:
Yeah, this should be changed when merging the PR. This URL was used only for QA on different staging server (the main one was used by other feature team)